src/parsers/trades/revolut/index.ts

```diff
@@ -0,0 +1,56 @@
+import { getCSVData } from '../../';
+import { EXCHANGES, IImport, IPartialTrade, ITrade } from '../../../types';
+import { createDateAsUTC, createID } from '../../utils';
+
+interface IRevolut {
+    Type: string;
+    Product: string;
+    'Started Date': string;
+    'Completed Date': string;
+    Description: string;
+    Amount: string;
+    Currency: string;
+    'Fiat amount': string;
+    'Fiat amount (inc. fees)': string;
+    Fee: string;
+    'Base currency': string;
+    State: string;
+    Balance: string;
+}
+
+export default async function processData(importDetails: IImport): Promise<ITrade[]> {
+    const data: IRevolut[] = await getCSVData(importDetails.data) as IRevolut[];
+    const internalFormat: ITrade[] = [];
+    for (const trade of data) {
+        if (trade.Type != 'EXCHANGE') {
+            console.log(`Not an exchange ${trade.Type}`);
+            continue;
+        }
+        const tradeToAdd: IPartialTrade = {};
+        tradeToAdd.date = createDateAsUTC(new Date(trade['Completed Date'])).getTime();
+        tradeToAdd.exchange = EXCHANGES.Revolut;
+        const bought = trade.Description.substr(13);
```

Comment:
Is there a better way we could pull out which currency was bought/sold? Do you have an example description I could look at?

Reply:
I will send one to you

Reply:
[account-statement_2020-10-01_2022-11-30_en-gb_92e922.csv](https://github.com/starsoccer/cryptotithe/files/10130466/account-statement_2020-10-01_2022-11-30_en-gb_92e922.csv)

Reply:
Could we do something like this instead?
```
const bought = trade.Description.substr(trade.Description.lastIndexOf(' '));
```

This should get whatever the last few characters are and wont be as reliant on a magic number

Reply:
Sure, let’s do that

Reply:
Please check my force-push. I just added +1 otherwise the space would be taken as well.